k2/csrc/CMakeLists.txt

```diff
@@ -265,10 +269,19 @@ file(COPY
   DESTINATION
     ${PROJECT_BINARY_DIR}/include/k2
 )
+file(GLOB HEADER_LIST "*.h")
```

Comment:
@csukuangfj  I am not sure is it a good way to copy all the header files to install directory, what's your oponions?